GNUmakefile

```diff
@@ -1,4 +1,4 @@
-TEST?=$$(go list ./...)
+TEST?=$$(go list ./... | grep -v scripts)
```

Comment:
Can we filter on the repo-specific prefix like `github.com/hashicorp/terraform-provider-google/scripts/`? That'll minimise the chance of false positives in the future.

Reply:
Yep!